armada/cli/apply.py

```diff
@@ -39,6 +35,8 @@ def get_parser(self, prog_name):
                             help='Armada yaml file')
         parser.add_argument('--dry-run', action='store_true',
                             default=False, help='Run charts with dry run')
+        parser.add_argument('--trace', action='store_true',
```

Comment:
this is confusing, esp when there is a `--debug` option. would `--debug-logging` not be more appropriate?